src/main/resources/data/minecolonies/tags/items/mechanic_product.json

```diff
@@ -1,31 +1,32 @@
-{
-  "replace": false,
-  "values": [
-    "minecraft:blue_ice",
-    "minecraft:enchanting_table",
-    "minecraft:torch",
-    "minecraft:soul_torch",
-    "minecraft:jack_o_lantern",
-    "minecraft:lantern",
-    "minecraft:soul_lantern",
-    "minecraft:packed_ice",
-    "minecraft:tripwire_hook",
-    "#minecraft:rails",
-    "#forge:storage_blocks",
-    "minecraft:bone_block",
-    "minecraft:sea_lantern",
-    "minecraft:stone_pressure_plate",
-    "minecraft:polished_blackstone_pressure_plate",
-    "#minecraft:wooden_pressure_plates",
-    "minecraft:light_weighted_pressure_plate",
-    "minecraft:heavy_weighted_pressure_plate",
-    "minecraft:sticky_piston",
-    "minecraft:trapped_chest",
-    "minecraft:ender_chest",
-    "minecraft:lever",
-    "minecraft:daylight_detector",
-    "minecraft:dried_kelp_block",
-    "minecraft:honey_block",
-    "minecraft:honeycomb_block"
-  ]
-}
+{
+  "replace": false,
+  "values": [
+    "minecraft:blue_ice",
+    "minecraft:enchanting_table",
+    "minecraft:torch",
+    "minecraft:soul_torch",
+    "minecraft:jack_o_lantern",
+    "minecraft:lantern",
+    "minecraft:soul_lantern",
+    "minecraft:packed_ice",
+    "minecraft:tripwire_hook",
+    "#minecraft:rails",
+    "#forge:storage_blocks",
+    "minecraft:bone_block",
+    "minecraft:sea_lantern",
+    "minecraft:stone_pressure_plate",
+    "minecraft:polished_blackstone_pressure_plate",
+    "#minecraft:wooden_pressure_plates",
+    "minecraft:light_weighted_pressure_plate",
+    "minecraft:heavy_weighted_pressure_plate",
+    "minecraft:sticky_piston",
+    "minecraft:trapped_chest",
+    "minecraft:ender_chest",
+    "minecraft:lever",
+    "minecraft:daylight_detector",
+    "minecraft:dried_kelp_block",
+    "minecraft:honey_block",
+    "minecraft:honeycomb_block",
+    "minecraft:diamond"

```

Comment:
Why did you add diamonds here? I don't think other storage blocks can be uncrafted by the mechanic

Reply:
Do you think I should move that over to the blacksmith then?

Reply:
Yea blacksmith, to match with the rest